### DISCUSSION TITLE
Magisk Module not accessible on http://mitm.it/cert/magisk ("Not Found")

### DISCUSSION THREAD

**Discussion by EmilEmilchen** [mitmproxy/mitmproxy#7109] [Q&A] [answered]
The title is pretty self explanatory and I really don't know what to do anymore. I even tried packaging the magisk module myself but I keep getting "Unzip error" in the Magisk App:
![image](https://github.com/user-attachments/assets/ba358a29-644e-412d-ae2e-bd24324be777)

When accessing http://mitm.it/cert/magisk I get:
![image](https://github.com/user-attachments/assets/651462bb-4d3a-45d1-8cf8-213a48ad1aea)
However accessing http://mitm.it/ works just like expected:
![image](https://github.com/user-attachments/assets/754c2a6c-874f-4633-8746-94b4f25df061)
mitmproxy also shows all the requests:
![image](https://github.com/user-attachments/assets/87f92ba5-b764-4933-b51e-8687756ac83c)

I really dont know what to do, I even tried running mitmproxy in and not inside a docker. No luck with that either.

**Comment by EmilEmilchen** [comment, marked as answer]
I'd still love to know why the issue I described occured but I've managed to find a workaround using [https://github.com/lupohan44/TrustUserCertificates](https://github.com/lupohan44/TrustUserCertificates).

**Comment by mhils** [comment]
Works fine for me:
```
$ mitmdump --version
Mitmproxy: 11.0.0.dev (+17, commit be4ba11)
Python:    3.12.3
OpenSSL:   OpenSSL 3.2.1 30 Jan 2024
Platform:  macOS-14.6.1-arm64-arm-64bit
$ mitmdump
[19:42:32.834] HTTP(S) proxy listening at *:8080.
[19:42:45.626][[::1]:53683] client connect
[::1]:53683: GET http://mitm.it/cert/magisk
          << 200 OK 3.0k
[19:42:45.717][[::1]:53683] client disconnect
```
```
$ curl -x localhost:8080 http://mitm.it/cert/magisk
Warning: Binary output can mess up your terminal. Use "--output -" to tell
Warning: curl to output it to your terminal anyway, or consider "--output
Warning: <FILE>" to save to a file.
```

**Reply by EmilEmilchen** [reply]
It doesn't seem to for me:
```
emil ~ ➜ curl -x localhost:8080 http://mitm.it/cert/magisk                                                                                                                                                                                                              18ms 
<!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>
```
I assume this is just some weird configuration issue on my end or something I'm honestly too lazy too troubleshoot this right now. Thanks for the quick reply though, I'll just go with the Magisk module I mentioned for now.
